docs/cookbook/adding-a-new-email-template.md

```diff
@@ -361,7 +361,51 @@ Shopsys\FrameworkBundle\Model\Mail\MailTemplateConfiguration:
               - '@=service("App\\Component\\Mail\\PasswordChangedMailTemplateVariablesProvider").create()'
 ```
 
+## Bonus: Enable testing the email template in administration on click
+
+This step is not necessary for the email template to work, but with a little effort, it could add a great value for the developers, testers, and the e-shop administrators.
```

Comment:
```suggestion
This step is unnecessary for the email template to work, but with a little effort, it could add great value for the developers, testers, and e-shop administrators.
```